tests/infrastructure/vm_template/test_vm_template_admission.py

```
@@ -56,6 +64,32 @@ def test_invalid_by_default_template_rejected(self):
+                virtual_machine={
+                    "metadata": {"name": "${NAME}"},
+                    "spec": {
+                        "runStrategy": "Halted",
+                        "template": {
+                            "spec": {
+                                "domain": {"devices": {}},
+                            }
+                        },
+                    },
+                },
+            ):
```

Ruff (0.15.21)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)